client/my-sites/email/controller.js

```diff
@@ -113,6 +113,7 @@ export default {
 		pageContext.primary = (
 			<CalypsoShoppingCartProvider>
 				<EmailProvidersInDepthComparison
+					callbackPath={ pageContext.query.callbackPath }
```

Comment:
Thinking a bit more, what do you think of `referrer` or `origin` instead? `previousRoute` might be even better (note in this case we should use `previous-route` as url parameter instead of the cameCase notation).

Reply:
Guess what, there is a `getPreviousRoute()` selector that we could even use, and which would simplify our code 😀.